include/SPTPersistentCache/SPTPersistentCache.h

```diff
@@ -20,6 +20,17 @@
  */
 #import <Foundation/Foundation.h>
 
+#ifndef SPT_BUILDING_FRAMEWORK
+#define SPT_BUILDING_FRAMEWORK 0
+#endif
+#if SPT_BUILDING_FRAMEWORK
+//! Project version number for SPTDataLoader.
+FOUNDATION_EXPORT double SPTDataLoaderVersionNumber;
+
+//! Project version string for SPTDataLoader.
+FOUNDATION_EXPORT const unsigned char SPTDataLoaderVersionString[];
+#endif // SPT_BUILDING_FRAMEWORK
+
```

Comment:
@dflems Those exports are part of what Xcode uses for creating a Framework module and Swift compatibility. Exactly what Xcode uses them for is beyond my knowledge. However we don’t want them for our static library.